DataScience/loggers/logger_wrapper.py

```diff
@@ -0,0 +1,41 @@
+from fluent import asyncsender as sender
+import json
+import sys
+import traceback
+from loggers.geneva_logger import GenevaLogger 
+from loggers.console_logger import ConsoleLogger
+
+class Logger:
+    loggers = []
+
+    @classmethod
+    def create_loggers(cls, geneva=False, **geneva_args):
```

Comment:
Isn't the `geneva` boolean redundant information? I would suggest `geneva_args` be renamed to `args` (these args could be for any logger) and the existence of a `--geneva***` arg or a `--logger_type=geneva` as the indicator of whether or not a geneva logger should be added.

Reply:
fixed